pkg/centos7/Dockerfile

```diff
@@ -4,32 +4,57 @@
 # The resulting image is ready to run the pyinstaller on container start and drop hubble<version>-coreos.tar.gz
 # in the /data directory. Mount /data volume into a directory on the host to access the package.
 # To run the container:  docker run -it --rm -v `pwd`:/data <image_name>
+# Requires docker 17.05 or higher
 
+# Set this argument to "local" if you want to build osquery for local code.
+# In that case, osquery folder must exist besides Dockerfile
+ARG OSQUERY_BUILD_ENV=remote
+
+#--------------- TEMP CONTAINER FOR LOCAL OSQUERY -------------------------
+FROM alpine as osquery_local
+ONBUILD COPY osquery /osquery
+ONBUILD RUN echo "Copying osquery from local folder"
+
+
+
+#--------------- TEMP CONTAINER FOR GIT OSQUERY ----------------------------
+FROM alpine/git as osquery_remote
+ENV OSQUERY_SRC_VERSION=0314871908273b2770129da9f5a7c3206f224f68
```

Comment:
In dev Dockerfiles, OSQUERY_SRC_VERSION's default value is set to "3.3.2".
I can make the same change here also. 
But since the osquery version bump up may get obscured in the other code change of this PR, I have not changed it for now.

Let me know if I should change the version to "3.3.2" here as well.
Just want to make sure that you are aware of this version change before I do so.

Reply:
This is something for @fossam to decide. I think the answer is yes, use 3.3.2, but he has more insight into osquery than I do.